script/installation/packages.sh

```diff
@@ -88,13 +133,14 @@ if [ "$DISTRO" = "UBUNTU" ]; then
         $PKG_CLANG \
         git \
         g++ \
+        clang-3.7 \
+        $CMAKE_NAME \
+        libgflags-dev \
```

Comment:
I think we already have $PKG_CMAKE, $PKG_CLANG, and libgflags-dev out there in the list. Those three here seem duplicated?